### ISSUE TITLE
Save the status of the realizations in the runpath

### ISSUE THREAD

**Issue by hnformentin** [equinor/ert#8594]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when I close the gui and loose the overview of the realizations/iterations status. I also saw some colleagues saving screenshots of the view and trying to squeeze it for fitting in a power point presentation. That would also help when there are bugs in the user interface, then it would be more simple to have the information available.

**Describe the solution you'd like**
It would be great if the overview of the realization status was saved automatically in the run path, such as logs are. Could be figure and an excel friendly format, for example.

**Describe alternatives you've considered**

![image](https://github.com/user-attachments/assets/251364a3-f623-4885-8e14-fd73ba6db8eb)

**Comment by xjules** [comment]
~~This can be part of the prior ensemble (the one actually running), in the transition data folder (see. parent issue for more details).~~ Please see the comments bellow.
We can always just overwrite the last full snapshot event.

**Comment by jonathan-eq** [comment]
It would be sufficient to dump a FullSnapshotEvent to storage after every merge with SnapshotUpdateEvent

**Comment by xjules** [comment]
> It would be sufficient to dump a FullSnapshotEvent to storage after every merge with SnapshotUpdateEvent

Yes, that should suffice. Also as discussed, we should make an entry on the experiment level (storage/exp/status) where the snapshot will reside.
